web/src/popover_menus.js

```diff
@@ -272,6 +280,113 @@ export function initialize() {
         },
     });
 
+    tippy_no_propagation("#stream_filters .topic-sidebar-menu-icon", {
+        ...left_sidebar_tippy_options,
+        onShow(instance) {
+            popover_instances.topics_menu = instance;
+            on_show_prep(instance);
+            const elt = $(instance.reference).closest(".topic-sidebar-menu-icon").expectOne()[0];
+            const args = popover_menus_data.get_topic_popover_content_context(elt);
+            instance.setContent(parse_html(render_topic_sidebar_actions(args)));
+        },
+        onMount(instance) {
+            const $popper = $(instance.popper);
+            const stream_id = stream_popover.elem_to_stream_id(
+                $(instance.reference)
+                    .closest(".topic-sidebar-menu-icon")
+                    .closest(".narrow-filter")
+                    .expectOne(),
+            );
+            const topic_name = $(instance.reference)
+                .closest(".topic-sidebar-menu-icon")
+                .closest("li")
+                .expectOne()
+                .attr("data-topic-name");
+            $popper.one("click", ".sidebar-popover-mute-topic", (e) => {
+                if (!stream_id) {
+                    return;
+                }
+
+                muted_topics_ui.mute_topic(stream_id, topic_name);
+                e.stopPropagation();
+                e.preventDefault();
```

Comment:
I'm wondering if we truly need to use `preventDefault` in this and other click handlers. If not, maybe we could remove them?

Reply:
Removed it.